### PR TITLE
Replace individual authors on Zenodo with team name 

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR proposes to update the zenodo.json file containing the meta-data of the [Zenodo publication](https://zenodo.org/record/6402507#.YmGa6tpBxPZ) of the software.

### CODE REVIEW

**Review by terrymyc — APPROVED** [review]

File: .zenodo.json [hunk 1]
```diff
@@ -1,85 +1,10 @@
 {
-	"description": "The Automated Systematic Review (ASReview) project implements learning algorithms that interactively query the researcher. This way of interactive training is known as Active Learning. ASReview offers support for classical learning algorithms and state-of-the-art learning algorithms like neural networks. The software ships with an user interface as well as a simulation mode.",
-	"title": "ASReview: Active learning for systematic reviews",
+	"description": "The Automated Systematic Review (ASReview) project implements learning algorithms that interactively query the researcher. This way of interactive training is known as Active Learning. ASReview offers support for classical learning algorithms and state-of-the-art learning algorithms like neural networks. ASReview LAB is ASReview LAB is the graphical user interface of the open-source research software and ships with an Oracle, Exploration and Simulation Mode.",
```
Comment (J535D165) [inline .zenodo.json:2]:
```suggestion
	"description": "The Automated Systematic Review (ASReview) project implements learning algorithms that interactively query the researcher. This way of interactive training is known as Active Learning. ASReview offers support for classical learning algorithms and state-of-the-art learning algorithms like neural networks. ASReview LAB is the graphical user interface of the open-source research software and ships with an Oracle, Exploration and Simulation Mode.",
```

Comment (J535D165) [inline .zenodo.json:2]:
```suggestion
	"description": "The Automated Systematic Review (ASReview) project implements learning algorithms that interactively query the researcher. This way of interactive training is known as Active Learning. ASReview offers support for classical learning algorithms and state-of-the-art learning algorithms like neural networks. ASReview LAB is the graphical user interface of the open-source research software and ships with an Oracle, Exploration and Simulation Mode.",
```

File: .zenodo.json [hunk 2]
```diff
@@ -1,85 +1,10 @@
...
+	"title": "ASReview LAB: Active learning for systematic reviews software",
```
Comment (J535D165) [inline .zenodo.json:3]: Would be nice to align with (the title bar of the tab)
https://github.com/asreview/asreview/blob/623434b022ae0772584c2dbcc0d6ab9492c190f0/asreview/webapp/public/index.html#L11

Comment (J535D165) [inline .zenodo.json:3]: Would be nice to align with (the title bar of the tab)
https://github.com/asreview/asreview/blob/623434b022ae0772584c2dbcc0d6ab9492c190f0/asreview/webapp/public/index.html#L11

Comment (Rensvandeschoot) [inline .zenodo.json:3]:
```suggestion
	"title": "ASReview LAB - A tool for AI-assisted systematic reviews",
```

Comment (Rensvandeschoot) [reply .zenodo.json:3]: do we want to call it a 'tool'? or Software? If the latter, then we need to update https://github.com/asreview/asreview/blob/623434b022ae0772584c2dbcc0d6ab9492c190f0/asreview/webapp/public/index.html#L11

Comment (terrymyc) [reply .zenodo.json:3]: I am fine with "tool".

**Pull request merged**